packages/nuxt/src/plugins/plugin.ts

```diff
@@ -125,18 +127,23 @@ function buildTarget(
   namedInputs: {
     [inputName: string]: any[];
   },
-  buildOutputs: string[]
+  buildOutputs: string[],
+  projectRoot: string
 ) {
   const targetDefaults = readTargetDefaultsForTarget(
     'build',
     context.nxJsonConfiguration.targetDefaults,
     '@nx/nuxt:build'
   );
 
-  const targetConfig: TargetConfiguration<Partial<NuxtBuildExecutorOptions>> = {
-    executor: '@nx/nuxt:build',
-    defaultConfiguration: 'production',
+  const targetConfig: TargetConfiguration = {
+    executor: 'nx:run-commands',
+    options: {
+      command: 'npx nuxi build',
```

Comment:
You should use `detectPackageManager` + `getPackageManagerCommand` instead of hardcoded `npx`. This would fail with `PnP`